functions_advanced/task3.js

```diff
@@ -0,0 +1,40 @@
+function divide(numerator, denominator) {
```

Comment:
add checks for arguments type, using operator `typeof`

Reply:
since isNaN returns false for numeric strings
![image](https://github.com/viaylya1/aqa-advanced/assets/53534240/4ddbd011-3145-4847-b2b2-303e4989dc01)

Reply:
Thanks for pointing it out! I added also typeof checks for the arguments